pyproject.toml

```diff
@@ -36,6 +36,8 @@ dependencies = [
   "requests",
   "rich ~= 13.0",
   "rfc8785 ~= 0.1.2",
+  # NOTE(dm): Under very active development, so stricly pinned.
+  "rfc3161-client ~= 0.1.0",
```

Comment:
Comment says this is strictly pinned, but it isn't -- did you mean `==` here?

Reply:
Related, since the `0.1.0` release was a mistake, we could push the `0.0.1` release to PyPI and yank `0.1.0`